modules/dfpAdServerVideo.js

```diff
@@ -103,6 +107,92 @@ export default function buildDfpVideoUrl(options) {
   });
 }
 
+export function notifyTranslationModule(fn) {
+  fn.call(this, 'dfp');
+}
+
+getHook('registerAdserver').before(notifyTranslationModule);
```

Comment:
Do we need to update the `registerAdserver` function in the `categoryTranslation` file to support the DFP adserver?

Further if someone is trying to load an instream video bid with DFP, this hook (with where it's currently positioned in the base file) will attempt to setup the categoryTranslation function and load the appropriate json file.  If there is no previous localStorage translation data, the module will attempt to load an `undefined` url that results in a 404 error in the console.

If we can't restrict adding this hook to only when a request for an adpod bid is made (not sure if that's doable/practical off the top of my head), then we should add contingencies to the category code to abort its look-up to prevent the above scenario.

Reply:
Yes, updated the function in categoryTranslation module to not initialize translation in case of dfp as we do not have default mapping file for DFP adserver.
In DFP, labels are used to manage competitive exclusion. Publisher can define any string value as label and hence it is not possible to have default mapping file for DFP. 

With the latest commit initTranslation will only execute if freewheel is adserver.

Also I see that undefined in targeting value which is coming from here https://github.com/prebid/Prebid.js/blob/master/modules/adpod.js#L216. We need return statement as we are rejecting bid here